core/mpc/keysign/txInputData/cosmos/index.ts

```
@@ -94,7 +94,38 @@ export const getCosmosTxInputData: TxInputDataResolver<'cosmos'> = ({
+                  executeMsg: formattedMessage,
+                  coins: contractPayload.coins.map(c => {
+                    return TW.Cosmos.Proto.Amount.create({
+                      denom: c.contractAddress.toLowerCase(),
+                      amount: keysignPayload.toAmount,
+                    })
+                  }),
+                }),
+            }),
+          ],
+        }
+      } else if (memo && memo.startsWith('merge:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)